aries-site/src/examples/components/menu/MenuIconExample.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { Box, List, Menu } from 'grommet';
+import { More } from 'grommet-icons';
+
+export const MenuIconExample = () => {
+  const data = ['User 1', 'User 2', 'User 3'];
+
+  return (
+    <Box fill>
```

Comment:
Do we want a maxWidth on Box? On wide screens the example gets stretched out pretty far.